File: caddisfly-app/app/src/androidTest/java/org/akvo/UserInterfaceSuite.java
package org.akvo;

import org.akvo.caddisfly.diagnostic.DiagnosticTest;
import org.akvo.caddisfly.test.CbtTest;
import org.akvo.caddisfly.ui.IntroTest;
import org.junit.runner.RunWith;
import org.junit.runners.Suite;

// Runs all unit tests.
@RunWith(Suite.class)
@Suite.SuiteClasses({DiagnosticTest.class,
        CbtTest.class,
        IntroTest.class})
public class UserInterfaceSuite {
}

File: caddisfly-app/app/src/androidTest/java/org/akvo/UserInterfaceSuite.kt
package org.akvo

import org.akvo.caddisfly.misc.MiscTest
import org.akvo.caddisfly.test.CbtSurveyTest
import org.akvo.caddisfly.test.TesterSwatchSurveyTest
import org.akvo.caddisfly.ui.IntroTest
import org.junit.runners.Suite

// Uncomment @Runwith below to run this suite. Commented because gradlew runs the tests twice
// https://github.com/gradle/gradle/issues/2603
//@RunWith(Suite::class)
@Suite.SuiteClasses(MiscTest::class, CbtSurveyTest::class, TesterSwatchSurveyTest::class, IntroTest::class)
class UserInterfaceSuite
